lib/main_common.pm

```diff
@@ -1420,6 +1420,13 @@ sub load_extra_tests_y2uitest_cmd {
     loadtest 'yast2_cmd/yast_sysconfig';
     loadtest 'yast2_cmd/yast_keyboard';
     loadtest 'yast2_cmd/yast_nfs_server';
+
+    #temporary runs for QAM while tests under y2uitest_ncurses are being ported
```

Comment:
if its temporary for QAM then you can add condition for only QAM tests and drop if is_sle

Reply:
hello, all the tests that I have added are 'opensuse prof', they should work out of the box.
The only exclusion in fact is NIS - I just replicated its current status from y2uitest_ncurses, which also states:

#internal nis server in suse network is used, but this is not possible for openqa.opensuse.org
        loadtest "console/yast2_nis" if is_sle;

danke!